### DISCUSSION TITLE
Is FerretDB thread-safe?

### DISCUSSION THREAD

**Discussion by Andrew-S-Rosen** [FerretDB/FerretDB#3725] [General]
Is FerretDB safe to use when running parallel, multi-threaded routines in Python? In other words, is there some sort of locking mechanism in place to prevent race conditions?

**Comment by Andrew-S-Rosen** [comment]
Sorry, didn't mean to open this one since there's no Python API.

**Reply by AlekSi** [reply]
Well, there is: https://github.com/mongodb/mongo-python-driver or any other MongoDB driver for Python
